ci/repo-config/macos/alibuildmac03/alidist-o2.env

```diff
@@ -9,7 +8,7 @@ ALIBUILD_DEFAULTS=o2
 TRUSTED_USERS=
 TRUST_COLLABORATORS=true
 DONT_USE_COMMENTS=1
-DEVEL_PKGS="$PR_REPO $PR_BRANCH $PR_REPO_CHECKOUT
+DEVEL_PKGS="$PR_REPO $PR_BRANCH
 AliceO2Group/AliceO2 dev O2
-alisw/alidist master"
```

Comment:
Why do we remove `alisw/alidist master` here?

Reply:
Because it's already included due to the `$PR_REPO $PR_BRANCH` line.